docs/frontend-api/authentication.md

```diff
@@ -0,0 +1,112 @@
+# Authorization
+
+In a modern web application (or native mobile application), it’s likely that you’ll need to make parts of your app private so that not just anyone on the internet can access everything in your app.
+But if you want some users to be able to access those restricted parts, this requires implementing some form of user authentication to your app, so that your users can create an account, and later log back into that account, to access your protected content.
+The implementing some form of user authentication in the shopsys framework use access tokens.
+
+Access Tokens are used in token-based authentication to allow an application to access an API. For example, a customer orders API method can read orders for only one (authorized) customer.
+Once an application has received an Access Token, it will include that token as a credential when making API requests. To do so, it should transmit the Access Token to the API as a Bearer credential in an HTTP Authorization header.
+
+[JWT](https://jwt.io/introduction/) tokens are used as access tokens in the shopsys framework.
+
+You can read more about access tokens in [this](https://auth0.com/docs/tokens/concepts/access-tokens) article.
+
+The access token has a short lifetime (15 minutes in the base).
+So that users do not have to log in again after such a short time, the Shopsys framework introduces implementation of refresh tokens that have a longer lifetime (14 days in the base).
```

Comment:
```suggestion
So that users do not have to log in again after such a short time, the Shopsys Framework introduces the implementation of refresh tokens that have a longer lifetime (14 days in the base).
```